packages/cli/Command.js

```diff
@@ -28,7 +28,7 @@ export default class Command {
       throw new Error('Command is an *abstract* class')
     }
 
-    this.command = definition.name
```

Comment:
@mphstudios I thought it was confusing to have "command" being used to reference both the command object and command name, so I renamed this property `name`